src/Syncer/Command/SyncTimings.php

```diff
@@ -103,21 +115,28 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         foreach ($workspaces as $workspace) {
             $detailedReport = $this->reportsClient->getDetailedReport($workspace->getId());
+            $this->clients = array_merge($this->clients, $this->retrieveClientsForWorkspace($workspace->getId()));
+            $this->projects = array_merge($this->projects, $this->retrieveProjectsForWorkspace($workspace->getId()));
```

Comment:
`$this->projects` Should not be overridden here. This is an array coming from a config file. If you change the data and need to reuse it in another function. It will have unexpected behavior.